templates/apps/sumo-aws-apps/security-hub-org/security-hub-org-main.yaml

```diff
@@ -497,7 +436,7 @@ Resources:
     Condition: install_sumo_security_hub_app
     Properties:
       ServiceToken: !Ref ParentHelperFunctionArn
-      Region: !Select [0, !Split [",", !Ref RegionsToEnable]]
+      Region: !Ref "AWS::Region"
```

Comment:
This should be  security tooling region right ?

Reply:
i think use SecurityHubHomeRegion